README.md

```diff
@@ -18,6 +30,8 @@ Table of contents
     * [Model inference](#model-inference)
     * [Generate query and candidate vectors](#generate-query-and-candidate-vectors)
     * [Candidate ranker and assembling vector representations](#candidate-ranker-and-assembling-vector-representations)
+    * [Candidate ranking on-the-fly](#candidate-ranking-on-the-fly)
```

Comment:
There's no section `Candidate ranking on-the-fly`, the link does not work.

Reply:
Added this section for now as a placeholder: https://github.com/Living-with-machines/DeezyMatch/commit/537840a2905ce75fd76f075348c17a87edfedc66

I will continue working on the README later (updated the first ticket on this PR)